template/koa-rest-api/src/testing/types.ts

```diff
@@ -8,7 +8,13 @@ export type IdDescription = z.infer<typeof IdDescriptionSchema>;
 export const IdDescriptionSchema = z.union([
   z.object({
     id: z.string(),
-    description: z.string(),
+    description: z.union([
+      z.string(),
+      z.object({
+        fontSize: z.number(),
+        content: z.string(),
+      }),
```

Comment:
Sorry for the delay @JasonZzy0528! There doesn't appear to be a nested union here though; do you have a newer version of this file locally?

Reply:
hmm...I am a bit confused, the `IdDescription` is a union and its `description` field is a union, is `IdDescription` a nested union?

Reply:
Ah I misread, sorry about that 😅